### DISCUSSION TITLE
Can I resell this

### DISCUSSION THREAD

**Discussion by boringcommerce** [plausible/analytics#3006] [Self-Hosted Support]
Hello!

Is it possible for me to install this on my server, and create small extentions on this in order to be sold under my brand name?

**Comment by jesperordrup** [comment]
I dont know the answer to your question but I have a comment. I would keep the name, its a brand, its trusted for its values. If you rename it then you start over. Also why? Its cool to give back by keeping the name. We use plausible, we dont resell it but we sell and devop the servers to run the solutions. I would say that that is ok

**Comment by jezao** [comment]
According to the license, you can do whatever you want as you keep your solution Open Source.
